library/alloc/src/vec/mod.rs

```diff
@@ -2809,9 +2809,9 @@ impl<T, A: Allocator> From<Box<[T], A>> for Vec<T, A> {
     /// let b: Box<[i32]> = vec![1, 2, 3].into_boxed_slice();
     /// assert_eq!(Vec::from(b), vec![1, 2, 3]);
     /// ```
+    #[inline]
```

Comment:
Is there any particular reason to use `#[inline]` here? We generally avoid using it in the standard library unless a benchmark shows that it is necessary.

Reply:
No, just added it because this function immediately delegates to another one. I'll remove it.